Plugins/Public/base_plugin/Main.cpp

```diff
@@ -2624,39 +2635,6 @@ bool ExecuteCommandString_Callback(CCmds* cmd, const wstring &args)
 		}
 
 
-		return true;
-	}
-	else if (args.find(L"basedespawn") == 0)
```

Comment:
Aren't people using this? Did you mean to get rid of one of the baserespawn handlers instead?